src/Core/Algorithms/Legacy/Forward/BuildBEMatrixAlgo.cc

```diff
@@ -523,15 +519,15 @@ void BuildBEMatrixBase::make_auto_G_allocate(VMesh* hsurf, DenseMatrixHandle &h_
 }
 
 void BuildBEMatrixBase::make_auto_G(VMesh* hsurf, DenseMatrixHandle &h_GG_,
-double in_cond, double out_cond, double op_cond, const std::vector<double>& avInn)
+double in_cond, double out_cond, const std::vector<double>& avInn)
 {
   make_auto_G_allocate(hsurf, h_GG_);
-  BuildBEMatrixBaseCompute::make_auto_G_compute(hsurf, *h_GG_, in_cond, out_cond, op_cond, avInn);
+  BuildBEMatrixBaseCompute::make_auto_G_compute(hsurf, *h_GG_, in_cond, out_cond, avInn);
 }
 
 template <class MatrixType>
 void BuildBEMatrixBaseCompute::make_auto_G_compute(VMesh* hsurf, MatrixType& auto_G,
-  double in_cond, double out_cond, double op_cond, const std::vector<double>& avInn)
+  double in_cond, double out_cond, const std::vector<double>& avInn)
 {
   //const double mult = 1/(2*M_PI)*((out_cond - in_cond)/op_cond);  // op_cond=out_cond for all the surfaces but the outermost surface which in op_cond=in_cond
   const double mult = 1/(4*M_PI)*(out_cond - in_cond);  // op_cond=out_cond for all the surfaces but the outermost surface which in op_cond=in_cond
```

Comment:
This is the form that is is the matlab code in the FwdInvToolkit, which as far as I know is the basis of this module

Reply:
I'm not sure where the op_cond comes from.  seems like a good idea to remove it